contracts/HATKlerosConnector.sol

```diff
@@ -0,0 +1,490 @@
+        loserAppealPeriodMultiplier = _loserAppealPeriodMultiplier;
+    }
+
+    // ******************** //
+    // *    Governance    * //
+    // ******************** //
+
+    /** @dev Changes winnerMultiplier variable.
+     *  @param _winnerMultiplier The new winnerMultiplier value.
+     */
+    function changeWinnerMultiplier(uint256 _winnerMultiplier) external onlyGovernor {
+        winnerMultiplier = _winnerMultiplier;
+    }
+
+    /** @dev Changes loserMultiplier variable.
+     *  @param _loserMultiplier The new winnerMultiplier value.
+     */
+    function changeLoserMultiplier(uint256 _loserMultiplier) external onlyGovernor {
+        loserMultiplier = _loserMultiplier;
+    }
+
+    /** @dev Changes loserAppealPeriodMultiplier variable.
+     *  @param _loserAppealPeriodMultiplier The new loserAppealPeriodMultiplier value.
+     */
+    function changeLoserAppealPeriodMultiplier(uint256 _loserAppealPeriodMultiplier) external onlyGovernor {
+        loserAppealPeriodMultiplier = _loserAppealPeriodMultiplier;
+    }
+
+    /** @dev Update the meta evidence used for disputes.
+     *  @param _metaEvidence URI of the new meta evidence.
+     */
+    function changeMetaEvidence(string calldata _metaEvidence) external onlyGovernor {
+        metaEvidenceUpdates++;
+        emit MetaEvidence(metaEvidenceUpdates, _metaEvidence);
+    }
+
+    // ******************** //
+    // *    Challenges    * //
+    // ******************** //
+    
+    /** @dev Notify KlerosArbitrator that expert's committee decision was challenged. Can only be called by Hat arbitrator.
+     *  Requires the arbitration fees to be paid.
+     *  @param _claimId The Id of the active claim in Vault contract.
+     *  @param _evidence URI of the evidence to support the challenge.
+     *  Note that the validity of the claim should be checked by Hat arbitrator.
+     *  @return surplusFee The amount of fee that exceeds required arbitration cost.
+     */
+    function notifyArbitrator(bytes32 _claimId, string calldata _evidence) external payable returns (uint256 surplusFee) {
+        require(msg.sender == address(hatArbitrator), "Wrong caller");
+        require(!claimChallenged[_claimId], "Claim already challenged");
+
+        uint256 arbitrationCost = getArbitrationCost();
+        require(msg.value >= arbitrationCost, "Should pay the full deposit.");
+        surplusFee = msg.value - arbitrationCost;
+
+        claimChallenged[_claimId] = true;
+
+        uint256 localDisputeId = disputes.length;
+
+        DisputeStruct storage dispute = disputes.push();
+        dispute.claimId = _claimId;
+
+        // Preemptively create a new funding round for future appeals.
+        dispute.rounds.push();
+
+        uint256 externalDisputeId = klerosArbitrator.createDispute{value: arbitrationCost}(RULING_OPTIONS,  arbitratorExtraData);
+        dispute.externalDisputeId = externalDisputeId;
+        externalIDtoLocalID[externalDisputeId] = localDisputeId;
```

Comment:
I feel like I aksed this before, but can you remind me why we cannot use the externalDisputeId as a local reference as well?